stories/General/TagStory.tsx

```diff
@@ -1,81 +1,294 @@
-  <StorybookComponent
-    title="Tag"
-    code="import { Tag } from 'glints-aries'"
-    propsObject={props}
-    usage={`<Tag theme="blue">
-  Software Engineer
-  <CloseIcon color="white" />
-</Tag>`}
-  >
-    <div style={{ display: 'flex' }}>
-      <div style={{ textAlign: 'center', marginRight: '2em' }}>
-        <h3>Block</h3>
-        <Tag theme="orange" block>
-          <span style={{ marginRight: '12px' }}>Software Engineer</span>
-          <CloseIcon color="white" />
+const TagTypeStory = () => {
+  const props = {
+    Tag: [
+      {
+        name: 'block',
+        type: 'boolean',
+        defaultValue: <code>false</code>,
+        possibleValue: <code>true | false</code>,
+        require: 'no',
+        description: 'Sets Tag into block style.',
+      },
+      {
+        name: 'outline',
+        type: 'boolean',
+        defaultValue: <code>false</code>,
+        possibleValue: <code>true | false</code>,
+        require: 'no',
+        description: 'Sets Tag into outline style.',
+      },
+      {
+        name: 'border',
+        type: 'string',
+        defaultValue: <code>solid</code>,
+        possibleValue: <code>solid | dashed | none</code>,
+        require: 'no',
+        description: "Sets Tag's border style.",
+      },
+    ],
+  };
+  return (
+    <StorybookComponent
+      propsObject={props}
+      usage={`<Tag>Default</Tag>
+<Tag block>Block</Tag>
+<Tag outline>Outline</Tag>
+<Tag block outline>Block and Outline</Tag>`}
+    >
+      <Heading>Types</Heading>
+      <TagContainer>
+        <Tag>Default</Tag>
+      </TagContainer>
+      <TagContainer>
+        <Tag block>Block</Tag>
+      </TagContainer>
+      <TagContainer>
+        <Tag outline>Outline</Tag>
+      </TagContainer>
+      <TagContainer>
+        <Tag block outline>
+          Block and Outline
         </Tag>
-      </div>
-      <div style={{ textAlign: 'center', marginRight: '2em' }}>
-        <h3>Default</h3>
-        <Tag theme="grey">
-          <span style={{ marginRight: '12px', color: 'black' }}>
-            Software Engineer
-          </span>
-          <CloseIcon color="black" />
-        </Tag>
-      </div>
-      <div style={{ textAlign: 'center', marginRight: '2em' }}>
-        <h3>Outline</h3>
-        <Tag theme="green" outline>
-          <span style={{ marginRight: '12px' }}>Software Engineer</span>
-          <CloseIcon color="#93bd49" />
+      </TagContainer>
+      <TagContainer>
+        <Tag border="dashed">Dashed border</Tag>
+      </TagContainer>
+      <TagContainer>
+        <Tag border="none">None border</Tag>
+      </TagContainer>
+    </StorybookComponent>
+  );
+};
+
+const ClickableTagStory = () => {
+  const usage = `<Tag>Non-clickable tag</Tag>
+<Tag startIcon={<AddIcon onClick={() => alert('Clicked!')} />}>Clickable icon</Tag>
+<Tag endIcon={<CloseIcon onClick={() => alert('Clicked!')} />}>Clickable icon</Tag>
+<Tag onClick={() => alert('Clicked!')}>Clickable tag</Tag>`;
```

Comment:
I don't see this type of clickable tag in the spec, only cursor becomes pointer when mouse is hovering?

Reply:
Those are the example of how to make the Tag be clickable. 
1. directly add onClick event on the icon you pass.
2.add onClick event on the Tag directly, then the whole tag will be clickable.